core/src/text/textBuffer.cpp

```diff
@@ -87,16 +87,23 @@ bool TextBuffer::addLabel(const std::string& _text, Label::Transform _transform,
     float inf = std::numeric_limits<float>::infinity();
     float x0 = inf, x1 = -inf, y0 = inf, y1 = -inf;
 
+    // Stroke width is normalized by the distance of the SDF spread, then scaled
+    // to a char, then packed into the "alpha" channel of stroke. The .25 scaling
+    // probably has to do with how the SDF is generated, but honestly I'm not sure
```

Comment:
Probably but good to maintain parity for now? :)

On Fri, Sep 25, 2015 at 3:35 PM, Peter Richardson <notifications@github.com>
wrote:

> In core/src/text/textBuffer.cpp
> <https://github.com/tangrams/tangram-es/pull/303#discussion_r40467539>:
>
> > @@ -87,16 +87,23 @@ bool TextBuffer::addLabel(const std::string& _text, Label::Transform _transform,
> >      float inf = std::numeric_limits<float>::infinity();
> >      float x0 = inf, x1 = -inf, y0 = inf, y1 = -inf;
> >
> > +    // Stroke width is normalized by the distance of the SDF spread, then scaled
> > +    // to a char, then packed into the "alpha" channel of stroke. The .25 scaling
> > +    // probably has to do with how the SDF is generated, but honestly I'm not sure
>
> I imagine we'll change that behavior in js so that 5px == 5px…
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram-es/pull/303/files#r40467539>.
>